cli/cmd/org/members.go

```diff
@@ -1,20 +1,218 @@
 package org
 
 import (
-	"fmt"
-
+	"errors"
+	"github.com/rilldata/rill/admin/client"
+	"github.com/rilldata/rill/cli/cmd/cmdutil"
 	"github.com/rilldata/rill/cli/pkg/config"
+	adminv1 "github.com/rilldata/rill/proto/gen/rill/admin/v1"
 	"github.com/spf13/cobra"
 )
 
 func MembersCmd(cfg *config.Config) *cobra.Command {
 	membersCmd := &cobra.Command{
 		Use:   "members",
 		Short: "Members",
-		Run: func(cmd *cobra.Command, args []string) {
-			fmt.Println("not implemented")
+	}
+	membersCmd.AddCommand(ListMembersCmd(cfg))
+	membersCmd.AddCommand(AddCmd(cfg))
+	membersCmd.AddCommand(RemoveCmd(cfg))
+	membersCmd.AddCommand(SetRoleCmd(cfg))
+	membersCmd.AddCommand(LeaveOrgCmd(cfg))
+
+	membersCmd.PersistentFlags().StringVar(&cfg.OrgOverride, "org", "", "Organization Name")
+	return membersCmd
+}
+
+func ListMembersCmd(cfg *config.Config) *cobra.Command {
+	membersCmd := &cobra.Command{
+		Use:   "list",
+		Short: "List Members",
+		RunE: func(cmd *cobra.Command, args []string) error {
+			sp := cmdutil.Spinner("Listing members...")
+			sp.Start()
+
+			orgName := cfg.Org()
+			if orgName == "" {
+				return errors.New("cannot not find default org")
+			}
+
+			client, err := client.New(cfg.AdminURL, cfg.AdminToken())
+			if err != nil {
+				return err
+			}
+			defer client.Close()
+			resp, err := client.ListOrgMembers(cmd.Context(), &adminv1.ListOrgMembersRequest{
+				Organization: orgName,
+			})
+			if err != nil {
+				return err
+			}
+			sp.Stop()
+
+			if len(resp.Users) == 0 {
+				cmdutil.WarnPrinter("No members found")
+				return nil
+			}
+			cmdutil.SuccessPrinter("Members list \n")
+			cmdutil.TablePrinter(toUserTable(resp.Users))
+
+			return nil
 		},
 	}
 
 	return membersCmd
 }
+
+func AddCmd(cfg *config.Config) *cobra.Command {
+	addCmd := &cobra.Command{
+		Use:   "add",
+		Short: "Add Member",
+		RunE: func(cmd *cobra.Command, args []string) error {
+			orgName := cfg.Org()
+			if orgName == "" {
+				return errors.New("cannot not find default org")
+			}
+			if len(args) != 2 {
+				return errors.New("invalid number of arguments (expected 2) - Email Role")
+			}
```

Comment:
Use `Args:  cobra.ExactArgs(2)` (see other commands in CLI for example). Applies to multiple commands